packages/sdk/src/internal/toriiQueryBuilder.ts

```
@@ -110,13 +128,32 @@ export class ToriiQueryBuilder<T extends SchemaType> {
 
     /**
      * Create a new builder instance with pagination settings
+     *
      */
     static withPagination<T extends Record<string, Record<string, any>>>(
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)